### PR TITLE
Prompt selection

### PR DESCRIPTION
**Pull request by Drake-AI**
**Please describe the purpose of this pull request.**
Add prompt list selection with memgpt configure.

**How to test**
Run memgpt configure and select the desired prompt. You can use your custom prompts inside MEMGPT_DIR/prompts.

**Have you tested this PR?**
Tested with 0.1.18

Prompt selection

**Related issues or PRs**
Not sure, i will take a look.

**Is your PR over 500 lines of code?**
Not.

**Additional context**
I think it is useful to have the option to choose the prompt.

### CODE REVIEW

**Comment by cpacker** [comment]
@sarahwooders maybe we should put this behind a `memgpt configure --advanced`?

Another potential issue here is the presets refactor, but I'm probably overthinking it and it's OK to do system prompt select that's not tied to function selection (at least for now).

File: memgpt/cli/cli.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ def run(
     human: str = typer.Option(None, help="Specify human"),
     model: str = typer.Option(None, help="Specify the LLM model"),
     preset: str = typer.Option(None, help="Specify preset"),
+    prompt: str = typer.Option(None, help="Specify prompt"),
```
Comment (cpacker) [inline memgpt/cli/cli.py:41]: This should probably say "Specify system prompt"

File: memgpt/cli/cli.py [hunk 2]
```diff
@@ -38,6 +38,7 @@ def run(
     human: str = typer.Option(None, help="Specify human"),
     model: str = typer.Option(None, help="Specify the LLM model"),
     preset: str = typer.Option(None, help="Specify preset"),
```
Comment (cpacker) [inline memgpt/cli/cli.py:40]: @sarahwooders example here - technically "preset" = "system prompt" + "function set"

So there's a slight conflict allowing people to also specify "system prompt", but maybe it's OK? Just overwrite the implicit choice of system prompt in preset?

File: memgpt/cli/cli.py (hunk 1 above)
Comment (Drake-AI) [reply memgpt/cli/cli.py]: Done.

File: memgpt/cli/cli.py (hunk 2 above)
Comment (Drake-AI) [reply memgpt/cli/cli.py:40]: Maybe it is better to allow users to make their custom presets with system prompt and function set? But it is out of the scope of this PR.

**Comment by Drake-AI** [comment]
Solved in this PR: https://github.com/cpacker/MemGPT/pull/420

**Pull request closed without merging**